src/extension.ts

```diff
@@ -12,12 +12,42 @@ function findStyleSheetCreateDeclarations(
     if (
       ts.isVariableDeclaration(node) &&
       node.initializer &&
-      ts.isCallExpression(node.initializer) &&
-      node.initializer.expression.getText() === "StyleSheet.create"
+      ts.isCallExpression(node.initializer)
     ) {
-      const objectLiteral = node.initializer.arguments[0];
-      if (ts.isObjectLiteralExpression(objectLiteral)) {
-        styles[node.name.getText()] = objectLiteral;
+      const callExpression = node.initializer;
+      const callText = callExpression.expression.getText();
+      
+      if (callText === "StyleSheet.create") {
+        const objectLiteral = callExpression.arguments[0];
+        if (ts.isObjectLiteralExpression(objectLiteral)) {
+          styles[node.name.getText()] = objectLiteral;
+        }
+      }
+      
```

Comment:
Remove this extra new line so that the `else if ` is on line 25 matching the rest of the file. Otherwise looks good